Infusion/Packets/Server/BoatMovingPacket.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Infusion.IO;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+namespace Infusion.Packets.Server
+{
+    internal sealed class BoatMovingPacket : MaterializedPacket
+    {
+        public uint Serial { get; set; }
+        public byte BoatSpeed { get; set; }
+        public Direction MovingDirection { get; set; }
+        public Direction FacingDirection { get; set; }
+        public List<PositionEntity> PositionEntities { get; set; }
```

Comment:
It is not a good practice to expose Lists as public members. Packets in Infusion usually expose arrays (for example CharactersStartingLocationsPacket) or IEnuerables (like AddMultipleItemsInContainerPacket).